libs/components/toast/src/lib/modules/toast/fixtures/toast-with-toaster-service.component.fixture.ts

```diff
@@ -8,7 +8,7 @@ import { SkyToasterService } from '../toaster.service';
 
 @Component({
   selector: 'sky-test-cmp',
-  templateUrl: './toast.component.fixture.html',
+  templateUrl: './toast-with-toaster-service.component.fixture.html',
```

Comment:
I found this when working... we actually had the `html` file but the fixture was pointing at the wrong one. I didn't research the existing tests any more but they still passed when I changed it. We may just not need this fixture (shrug)